android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -539,6 +547,38 @@ private void togglePlayerControlVisibility() {
     private void initializePlayerControl() {
         if (playerControlView == null) {
             playerControlView = new PlayerControlView(getContext());
+
+            // Apply TV specific Styling
+            if(isTelevision()){
+                // Controller padding to account for overscan
+                LinearLayout exoControllerLayout = playerControlView.findViewById(R.id.exo_controller);
```

Comment:
This approach can be a bit tricky if we decide to override the XML inside of the app. Is there a way that we can solve this with device relative sizes? Some of these changes might also be fine for mobile?